templates/default/event/eventadd.html

```diff
@@ -172,8 +182,20 @@
 </TABLE>
 </form>
 <SCRIPT type="text/javascript">
-<!--
```

Comment:
W dniu 15.03.2016 13:30, Maciej Wawryk napisał(a):
> In templates/default/event/eventadd.html [1]:
> 
>> @@ -172,8 +182,20 @@
>>  </TABLE>
>>  </form>
>>  <SCRIPT type="text/javascript">
>> -<!--
> 
> a co one dają ?

W niektórych dziwnych przeglądarkach nie wyświetla się wtedy kod 
javascript
ze znacznika <script>. Dzięki komentarzom blokowym.

> --
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/lmsgit/lms/pull/682/files#r56154590
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/lmsgit/lms/pull/682#discussion_r56154590

-- 
Pozdrawiam
Tomasz Chiliński, Chilan